jbpm-wb-common/jbpm-wb-common-client/src/main/java/org/jbpm/workbench/common/client/list/AbstractMultiGridPresenter.java

```diff
@@ -32,6 +32,9 @@
 
 public abstract class AbstractMultiGridPresenter<T extends GenericSummary, V extends MultiGridView> extends AbstractScreenListPresenter<T> implements RefreshSelectorMenuBuilder.SupportsRefreshInterval {
 
+    public static final String SEARCH_PARAMETER_PROCESS_INSTANCE_ID = "processInstanceId";
+    public static final String SEARCH_PARAMETER_PROCESS_DEFINITION_ID = "processDefinitionId";
```

Comment:
Perhaps the search parameter IDs could be defined in `PerspectiveIds`, with the IDs of the perspectives which receive them as parameters. What do you think?